### ISSUE TITLE
Update peerDependencies to core@^3.0.0

### ISSUE THREAD

**Issue by quoininc-huudatran** [stryker-mutator/stryker#2095]
**Summary**

The new version 3.0.0 still refers to 2.0.0 in the peerDependencies.

**Stryker config**

**Test runner config**

**Stryker environment**

```
├─┬ UNMET PEER DEPENDENCY @stryker-mutator/core@3.0.0
│ ├─┬ @stryker-mutator/api@3.0.0
│ ├── @stryker-mutator/util@3.0.0
├── @stryker-mutator/html-reporter@3.0.0
├─┬ @stryker-mutator/javascript-mutator@3.0.0
│ ├─┬ @stryker-mutator/api@3.0.0
├─┬ @stryker-mutator/mocha-framework@3.0.0
│ ├─┬ @stryker-mutator/api@3.0.0
│ └── @stryker-mutator/util@3.0.0
├─┬ @stryker-mutator/mocha-runner@3.0.0
│ ├─┬ @stryker-mutator/api@3.0.0
├─┬ @stryker-mutator/typescript@3.0.0
│ ├─┬ @stryker-mutator/api@3.0.0
│ ├── @stryker-mutator/util@3.0.0
npm ERR! peer dep missing: @stryker-mutator/core@^2.0.0, required by @stryker-mutator/javascript-mutator@3.0.0
npm ERR! peer dep missing: @stryker-mutator/core@^2.0.0, required by @stryker-mutator/mocha-framework@3.0.0
npm ERR! peer dep missing: @stryker-mutator/core@^2.0.0, required by @stryker-mutator/mocha-runner@3.0.0
npm ERR! peer dep missing: @stryker-mutator/core@^2.0.0, required by @stryker-mutator/typescript@3.0.0
npm ERR! peer dep missing: mocha@>= 2.3.3 < 7, required by @stryker-mutator/mocha-framework@3.0.0
npm ERR! peer dep missing: mocha@>= 2.3.3 < 7, required by @stryker-mutator/mocha-runner@3.0.0
├─┬ stryker-cli@1.0.0```

<!-- Please also add the test runner you are using.  Examples:-->


**Test runner environment**

<!-- Please add your test command here (probably the command you use in `npm test`) command -->


<!-- Please add any config files that are used by your test runner. For example jest.config.js, karma.conf.js, angular.json, mocha.opts, ...  -->



**Your Environment**

| software         | version(s)
| ---------------- | -------
| node             |
| npm              |
| Operating System |

**Add stryker.log**

<!-- Please add your stryker.log file. This file can be generated using `stryker run --fileLogLevel trace`. You can drag and drop it here. -->
<!-- Your source code is never logged to this file, however file names are logged. Feel free to obfuscate those log messages if you think it is a problem -->

**Comment by simondel** [comment]
😐 nice find

I’ll fix this today

**Comment by simondel** [comment]
This has been released as version 3.0.1 🎉
